_maps/RandomRuins/SpaceRuins/infested_frigate.dmm

```diff
@@ -0,0 +1,5162 @@
+/obj/effect/decal/cleanable/vomit/old,
+/obj/item/broken_bottle,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/turf/open/floor/mineral/plastitanium/red,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"VV" = (
+/obj/structure/cable,
+/obj/structure/barricade/security,
+/obj/structure/door_assembly/door_assembly_hatch{
+	name = "SYN-C Brutus Bridge"
+	},
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/turf/open/floor/mineral/plastitanium/red,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"Wk" = (
+/obj/structure/alien/weeds,
+/obj/structure/cable,
+/obj/effect/decal/cleanable/glass{
+	icon_state = "plasmatiny"
+	},
+/obj/effect/decal/cleanable/glass{
+	icon_state = "plasmalarge"
+	},
+/obj/item/shard{
+	icon_state = "titaniumtiny"
+	},
+/obj/structure/closet/crate/large{
+	anchored = 1;
+	name = "crew supplies"
+	},
+/obj/effect/spawner/random/aimodule/harmful,
+/obj/effect/spawner/random/contraband,
+/obj/effect/spawner/random/contraband,
+/obj/effect/spawner/random/contraband,
+/obj/effect/spawner/random/contraband,
+/obj/effect/spawner/random/decoration,
+/obj/effect/spawner/random/decoration,
+/obj/effect/spawner/random/decoration/paint,
+/obj/effect/spawner/random/entertainment/plushie_delux,
+/obj/effect/spawner/random/entertainment/toy_figure,
+/obj/machinery/nuclearbomb/beer,
+/obj/structure/sign/warning/electric_shock/directional/west,
+/obj/item/fuel_pellet,
+/obj/item/fuel_pellet,
+/obj/item/fuel_pellet,
+/obj/effect/spawner/random/exotic/languagebook,
+/obj/effect/spawner/random/entertainment/lighter,
+/obj/effect/spawner/random/entertainment/lighter,
+/obj/effect/spawner/random/special_lighter,
+/turf/open/floor/plating,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"WG" = (
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/obj/effect/decal/cleanable/blood/gibs{
+	icon_state = "floor7-old"
+	},
+/obj/effect/decal/cleanable/blood/gibs{
+	icon_state = "gibleg";
+	pixel_x = 3;
+	pixel_y = 19;
+	dir = 4
+	},
+/turf/open/floor/mineral/plastitanium/airless,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"WS" = (
+/obj/item/stack/rods{
+	amount = 50
+	},
+/turf/open/space/basic,
+/area/template_noop)
+"Xa" = (
+/obj/effect/turf_decal/tile/bar{
+	dir = 1
+	},
+/obj/effect/turf_decal/tile/bar,
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 1
+	},
+/obj/effect/spawner/random/trash/mess,
+/obj/structure/sign/poster/contraband/random/directional/south,
+/obj/item/kirbyplants/random,
+/obj/machinery/light/broken/directional/south,
+/turf/open/floor/mineral/titanium/tiled/white,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"Xc" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/table/reinforced,
+/obj/machinery/cell_charger,
+/obj/item/clothing/mask/facehugger/impregnated,
+/obj/machinery/power/apc/auto_name/directional/east,
+/obj/structure/cable,
+/obj/effect/spawner/random/exotic/antag_gear,
```

Comment:
I don't think you should be spawning in antag gear

Reply:
its not actual antag gear. this is fine

Reply:
I actually looked at the loot tables of these randomized items and they're what made me feel I could make this place zestier. The actual rolls here are pretty minor.